### DISCUSSION TITLE
What is the chemical symbol for gold ?

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#17] [Galaxy Qs] [answered]
What is the chemical symbol for gold ?

**Comment by arpittyagi102** [comment, marked as answer]
The chemical symbol for gold is **Au**.
It is derived from the Latin word Aurum, for Aurora, the goddess of dawn.

**Reply by Sriansh-raj** [reply]
Congratulations @arpittyagi102 you just earned a prestigious `Galaxy Brain` achievement 🥳

**Reply by Sriansh-raj** [reply]
@arpittyagi102  I have added your name to [Users_who_got_Galaxy_Brain.md](https://github.com/Sriansh-raj/GitHub-Achievements/blob/main/Users_who_got_Galaxy_Brain.md) 

Just verify if it is correct or not.
:)

**Reply by arpittyagi102** [reply]
Yes, its perfect !
Thank you
